client/src/components/HeaderTabs.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import { Box, Typography, Tabs, Tab } from "@material-ui/core";
+import { makeStyles, withStyles } from "@material-ui/core/styles";
+import NotificationsButton from "components/NotificationsButton";
+
+const useStyles = makeStyles((theme) => ({
+  tabContainer: {
+    marginLeft: theme.spacing(6),
+    marginRight: theme.spacing(6),
+    display: "flex",
+    alignItems: "center",
+  },
+  notificationIcon: {
+    top: "0",
+    right: "0",
+    height: "10px",
+    width: "10px",
+    backgroundColor: "#35d554",
+    borderRadius: "100%",
+    position: "absolute",
+  },
+  notificationButton: {
+    position: "relative",
+    padding: "5px 18px",
+    "&:hover": {
+      backgroundColor: "transparent",
+    },
+  },
+}));
+
+const StyledTab = withStyles({
+  root: {
+    fontWeight: "normal",
+    textTransform: "none",
+  },
+  selected: {
+    textShadow: "0 0 0.01px black",
+  },
+})(Tab);
+
+const HeaderTabs = (props) => {
+  const { selectedPage, setSelectedPage } = props;
+  const classes = useStyles();
+  const handleTabChange = (e, newValue) => setSelectedPage(newValue);
+
+  return (
+    <Box className={classes.tabContainer}>
+      <Tabs
+        value={selectedPage}
+        onChange={handleTabChange}
+        aria-label="page select"
+        TabIndicatorProps={{
+          style: { backgroundColor: "white" },
```

Comment:
let's change this to using a className

Reply:
Would [this](https://github.com/hatchways/team-apple-cider/pull/23/commits/ba7bb4de0dbee530f9f9628836b0119800dd2067) suffice, or does it need to be using className specifically?

Reply:
Yup, that looks fine!